lib/ParseSIL/ParseSIL.cpp

```diff
@@ -885,6 +887,75 @@ void SILParser::convertRequirements(SILFunction *F,
   }
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+/// Parse a SIL constant value (one of the following categories):
+/// - An integer literal (42).
+/// - A floating point literal (3.14).
+/// - A metatype (the instance type is parsed) ($Float).
+/// - An aggregate ([1, 2, 3]).
+/// Returns true on error.
+static bool parseSymbolicValue(SymbolicValue &value, SILParser &SP,
+                               SILBuilder &B) {
+  auto &P = SP.P;
+  auto &allocator = B.getModule().getASTContext().getAllocator();
+  // Handle integer literals.
+  if (P.Tok.is(tok::integer_literal)) {
+    APInt intValue;
+    if (SP.parseInteger(intValue, diag::sil_expected_graph_op_attr_value))
+      return true;
+    value = SymbolicValue::getInteger(intValue, allocator);
```

Comment:
I know I told you that we don't need types, but I think this is going to be problematic.  We actually do need to know the data type for integers and fp values :-(  Sorry about that.

How about syntax like:
[i64 42, f64 3.0]

Where you just handle i[digit]+  and f[digit]+, and special case f32/f64 for now?

Reply:
How about this syntax?
`[name1 42 : i64, name2 3.0 : f64]`

Reply:
Is it necessary to keep track of signed-ness (uint vs int)?

Reply:
Using "value : type" makes sense, but FYI the constants here have a different type system than Swift does, so please don't try to reuse the existing productions.